common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/CombinedIdentityAndEligibilityResponse.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.dhsc.htbhf.eligibility.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.dhsc.htbhf.dwp.model.v2.IdentityAndEligibilityResponse;
+
+/**
+ * The combined identity and eligibility response data from DWP and HMRC.
+ */
+
+@Value
+@Builder(toBuilder = true)
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class CombinedIdentityAndEligibilityResponse {
+
+    @JsonProperty("dwpIdentityAndEligibilityResponse")
+    private final IdentityAndEligibilityResponse identityAndEligibilityResponse;
+
+    @JsonProperty("hmrcHouseholdIdentifier")
+    private final String hrmcHouseholdIdentifier;
+
```

Comment:
Can we consider in-lining the IdentityAndEligibilityResponse? Maybe we should have a chat about it after stand-up

Reply:
Have inlined as we discussed.